### ISSUE TITLE
OK why are things made out of glowstone and obsidian ? BOP & This not working so good

### ISSUE THREAD

**Issue by crunkazcanbe** [GotoLink/Generatormods#3]
Go here https://copy.com/EieVvIDzGr78dCtf please , and have a look at these pictures I took of my world with this mod installed with BOP mod . Is it supposed to be this way? I found a underground city completely made from GlowStone yesterday to . in these pix it appears it is a underground city half way in a mountain and poking out of the mountain for a long ways .

Thank you for your help !

-Crunk

**Comment by GotoLink** [comment]
Yes, it is supposed to work this way, through the "UndergroundObsidianCity.tml" template file.
Removing the file or setting its relative "weight" to 0 will prevent this specific city from generating.

**Comment by crunkazcanbe** [comment]
Thanx man well this solved a lot of my problems . I love the underground cities , but not this particular one . Did you see how the under ground city was like half way out of the BOP mountain ? Is there away to make it for compatible with BOP ?

**Comment by GotoLink** [comment]
This is likely not a BOP issue. It can happen without any biome mod.
Depending on the city size and floor height, there is a chance an underground city would appear to break through a mountain.
You can set both values in each template, to further accommodate.
I personally like it this way, so i never bothered to "fix" this.
